lib/features/game/views/page_view_pages/question_page.dart

```diff
@@ -84,22 +90,41 @@ class QuestionInfoRow extends StatelessWidget {
     }
   }
 
+  Color questionDifficultyColor(QuestionDifficulty questionDifficulty) {
+    switch (questionDifficulty) {
+      case QuestionDifficulty.easy:
+        return KColors.basedYellowColor;
+      case QuestionDifficulty.medium:
+        return KColors.basedOrangeColor;
+      case QuestionDifficulty.hard:
+        return KColors.basedRedColor;
+      default:
+        return KColors.basedBlackColor;
+    }
+  }
+
   @override
   Widget build(BuildContext context) {
     return BlocBuilder<GamePageBloc, GamePageState>(
       builder: (context, gamePageState) {
-        return Row(
-          mainAxisAlignment: MainAxisAlignment.spaceAround,
-          children: [
-            Text(
-              'Question: ' + gamePageState.questionNumber.toString(),
-              style: TextStyle(fontSize: 15, color: KColors.basedBlackColor, fontWeight: FontWeight.bold),
-            ),
-            Text(
-              'Difficulty: ' + questionDifficultyToString(gamePageState.quizQuestion?.difficulty ?? QuestionDifficulty.medium),
-              style: TextStyle(fontSize: 15, color: KColors.basedBlackColor, fontWeight: FontWeight.bold),
-            ),
-          ],
+        return FittedBox(
+          fit: BoxFit.fitWidth,
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: [
+              Text(
+                'Question: ' + gamePageState.questionNumber.toString(),
+                style: TextStyle(fontSize: 15, color: KColors.basedBlackColor, fontWeight: FontWeight.bold),
+              ),
+              Text(
+                '  Difficulty: ' + questionDifficultyToString(gamePageState.quizQuestion?.difficulty ?? QuestionDifficulty.medium),
```

Comment:
remove whitespace! If you need padding use SizedBox and not spaces in string

Reply:
done